### ISSUE TITLE
Focus Builder: dropdown are not entirely visible (CSS issue)

### ISSUE THREAD

**Issue by Hugo-Prossaird** [mautic/mautic#14355]
### Mautic Series

5.2.x series

### Mautic installed version

5.2.0

### Way of installing

I'm using a Docker image from https://hub.docker.com/r/mautic/mautic

### PHP version

8.3

### What browsers are you seeing the problem on?

Chrome

### What happened?

In the Focus Builder, dropdown menus are not entirely visible when opened. The issue occurs because dropdown elements are clipped or confined within their parent container. Users are unable to view or interact with the entire menu.

<img width="516" alt="Capture d’écran 2024-12-09 à 15 02 35" src="https://github.com/user-attachments/assets/8d5d8d0d-9c6c-40be-b9df-3a06ed6874c4">

### How can we reproduce this issue?

1. Open the Focus Builder.
2. Navigate to any section where dropdown menus are present (e.g., Notification > placement).
3. Attempt to open a dropdown menu.

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by matbcvo** [comment]
PR #14349 fixes this issue.

**Comment by andersonjeccel** [comment]
Hm, @matbcvo great catch

I was trying to reproduce this bug right now with no success on that branch

**Comment by matbcvo** [comment]
Yes, I was able to reproduce the issue, and I noticed that your PR resolved it during testing.

**Comment by LordRembo** [comment]
@Hugo-Prossaird Can you confirm that this issue is fixed in the latest release? Then this can be closed.

**Comment by Hugo-Prossaird** [comment]
Now it's fixed ! ✅ 

<img width="509" alt="Capture d’écran 2025-01-07 à 08 42 25" src="https://github.com/user-attachments/assets/9abb0810-a40d-4ed5-9c1c-07683ba84163" />
